share/scudcloud.desktop

```diff
@@ -6,7 +6,6 @@ Categories=Network;Chat;
 Name=ScudCloud
 Comment=ScudCloud is an open-source client for Slack
 Keywords=slack;chat;instant-messaging;collaboration;
-Exec=/opt/scudcloud/scudcloud
+Exec=scudcloud
 StartupWMClass=scudcloud
-Icon=/opt/scudcloud/resources/scudcloud.png
-
+Icon=/usr/share/pixmaps/scudcloud.png
```

Comment:
It seems most .desktop files are `Icon=scudcloud.png` and expect the icon to be somewhere under `/usr/share/icons/<theme>/<size>/`  Also... there is no .png installed to the icon directories is there?  Its all svg IIRC.